### DISCUSSION TITLE
What does security mean?

### DISCUSSION THREAD

**Discussion by flg** [jmcollin78/versatile-thermostat-ui-card#36] [Q&A]
Hello,

I sometimes get this "security" warning over the UI card but I could not find how to interpret it:
![image](https://github.com/user-attachments/assets/e5de12fa-9216-4107-82b0-a8de7a1c2d3b)

What does this mean? Should we do something about it?

**Comment by DeadlyDeadOne** [comment]
From the Notes in feature-advanced.md; _the indoor and outdoor temperatures. Both must report values, or the thermostat will switch to "security" preset._

**Comment by Krzysztonek** [comment]
> What does this mean? Should we do something about it?

@flg ,
Here you will find [an answer on this particular topic](https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/en/feature-advanced.md#safety-mode).
